html/pfappserver/root/static/configurator/configuration.js

```diff
@@ -32,7 +32,8 @@ function saveStep(href) {
             data: {'general.domain': $('#general_domain').val(),
                    'general.hostname': $('#general_hostname').val(),
                    'general.dhcpservers': $('#general_dhcpservers').val(),
-                   'alerting.emailaddr': $('#alerting_emailaddr').val()}
+                   'alerting.emailaddr': $('#alerting_emailaddr').val(),
+                   'advanced.hash_passwords': $('input[name="advanced.hash_passwords"]').val()}
```

Comment:
All the other ones are using an html ID. Would be worth it to make it the same for all the fields in this section